src/network/http.js

```diff
@@ -0,0 +1,30 @@
+class http {
+  constructor(config) {
+    this.baseUrl = `http://${config.hostname}:${config.port}`;
+    this.header = {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    };
+  }
+
+  async post(path, body) {
+    const url = `${this.baseUrl}/${path}`;
+
+    const options = {
+      method: 'POST',
+      headers: this.header,
+      body: body && JSON.stringify(body),
+    };
+
+    return fetch(url, options)
+      .then(res => res.ok ? res.json() : this.handleError(res))
+      .catch(err => Promise.reject(err));
+  }
+
+  async handleError(res) {
+    switch (res.status) {
+      default:
+        return Promise.reject(Error(`Server returned status ${res.status} at ${res.url}`));
+    }
+  }
+}
```

Comment:
Also, you're forgetting to export HTTP :grimacing: 
```suggestion
}

export default HTTP
```

Reply:
Done!